robot/impl/local_robot.go

```diff
@@ -1183,6 +1184,37 @@ func (r *localRobot) reconfigure(ctx context.Context, newConfig *config.Config,
 
 	var allErrs error
 
+	// Maintenance config can be configured to block reconfigure based off of a sensor reading
+	// These sensors can be configured on the main robot, a fragment, or a remote
```

Comment:
how fragments and main robot config resolution works does not matter in how RDK works - the business logic there is in app and can change at any time, so we shouldn't document it here. while it does matter in terms of the end to end flow, all the RDK cares about is the config that ends up being passed to it, and that is what should be documented (so whether the sensor is on main or on remote)

Reply:
gotcha, ill remove the fragments blurb